src/main/steps/c100-rebuild/check-your-answers/content.ts

```diff
@@ -308,6 +315,8 @@ export const cyContent = {
     relationshipTo: 'Perthynas â',
     childLivingArrangements: "{firstname} {lastname}'s living arrangements (welsh)",
     whoDoesChildMainlyLiveWith: 'Who does {firstname} {lastname} mainly live with? (welsh)',
+    isOtherPersonAddressConfidential:
```

Comment:
Can you get the welsh translation from Pradeep and update.

Reply:
The welsh content is added now.